tests/test_html_field.py

```diff
@@ -0,0 +1,64 @@
+from django.contrib.auth.models import User
+from django.test import TestCase, Client
+
+import json
+from .testapp.models import Zoo, WebPage
+
+
+class HtmlFieldTestCase(TestCase):
+
+	def setUp(self):
+		super().setUp()
+		u = User(username='testuser', is_active=True, is_superuser=True)
+		u.set_password('test')
+		u.save()
+		self.client = Client()
+		r = self.client.login(username='testuser', password='test')
+		self.assertTrue(r)
+
+		self.zoo = Zoo(name='Apenheul')
+		self.zoo.save()
+
+		self.webpage = WebPage.objects.create(zoo=self.zoo, content='')
+
+	def test_save_normal_text_ok(self):
+		response = self.client.put(f'/web_page/{self.webpage.id}/', data=json.dumps({'content': 'Artis'}))
+		self.assertEqual(response.status_code, 200)
+
+	def test_simple_html_is_ok(self):
+		response = self.client.put(f'/web_page/{self.webpage.id}/',
+								   data=json.dumps({'content': '<h1>Artis</h1><b><p>Artis is a zoo in amsterdam</a>'}))
+		self.assertEqual(response.status_code, 200)
+
+	def test_wrong_attribute_not_ok(self):
+		response = self.client.put(f'/web_page/{self.webpage.id}/',
+								   data=json.dumps({'content': '<b onclick="">test</b>'}))
+		self.assertEqual(response.status_code, 400)
```

Comment:
Maybe also test the error code we get back from binder is correct? rather then only checking we get a 400 response. Because now we are not sure if the 400 is due to the reason we think.